test/tail.spec.js

```diff
@@ -0,0 +1,20 @@
+const assert = require('assert');
+const tail = require('../src/tailFunc');
+
+function test() {
+  const arr1 = [1, 2, 3, 4, 5];
+  const arr2 = [10, 20, 30, 40, 50];
+  const arr3 = [100, 200, 300, 400, 500];
+  const arr4 = [];
+
+  // success
+  assert.equal(tail(arr1), 5);
```

Comment:
do we expect 5 as result? did you run the tests?

Reply:
changed